uber/models.py

```diff
@@ -1182,14 +1182,14 @@ def last_first(cls):
     @property
     def watchlist_guess(self):
         try:
-            with sa.Session() as session:
+            with Session() as session:
                 return session.guess_attendee_watchentry(self)
```

Comment:
OK, so:

* Using `to_dict()` returns nothing (the page is blank).
* Using `.__dict__` returns several objects even though only one record exists. Also, none of the object's properties are accessible through the page (all the fields are blank).
* Expunging the object from the session also results in nothing.

However, using `return listify(self.watch_list) if self.watchlist_id else self.watchlist_guess` seems to work as we want it to. Is there any reason not to just do that?